libtransact/src/state/merkle/node.rs

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright 2021 Cargill Incorporated
```

Comment:
Should this include the Intel copyright from the previous file and a date range for Cargill?

Reply:
Fixed